environment.yml

```diff
@@ -6,15 +6,15 @@ dependencies:
   - python >=3.9
   - boltons >=20.1
   - bottleneck >=1.3.1
-  - cf_xarray >=0.6.1
+  - cf_xarray >=0.9.3
   - cftime >=1.4.1
   - click >=8.1
   - dask >=2.6.0
   - jsonpickle
   - numba
-  - numpy >=1.20.0
+  - numpy >=2
```

Comment:
Is there no way to maintain backwards compatibility, or is the effort not worth it?

Reply:
To maintain backwards compatibility, we need to to unpin pint and cf-xarray and accept that dimensionless units might be printed `""` or `"1"` depending on the version.

In both cases (empty string or `"1"`), xclim/cf-xarray would parse them as dimensionless so potential incompatibility wouldn't come from us.

We could even keep a CI run with numpy <2 to be sure. Some tests would need skipping as we now test explicitly for "1" in the dimensionless case.

Reply:
I would like to keep the older numpy around for a few versions in case we run into problems with other packages not adapting quickly enough. This gives us a bit of wiggle room on PAVICS.

Perhaps we could raise a warning when using older numpy to state that the outputs of dimensionless data will change when transitioning to numpy v2.0?

For the tests, we can make use of:
```python
from numpy import __version__ as __numpy_version__
from packaging.version import Parse

...

def test_test_test():
    out = _some_logic_here()

    if Parse(__numpy_version__) < Parse("2.0.0"):
        assert out.units == ""
    else:
        assert out.units == "1" 
```

Does that seem reasonable?

Reply:
Yes. And I updated the Version checks to check against `cf_xarray` instead, where the difference really comes from.